modules/web/src/de/diedavids/cuba/dataimport/web/importwizard/ImportWizard.groovy

```diff
@@ -236,9 +293,13 @@ class ImportWizard extends AbstractWindow {
     }
 
     void startImport() {
-        ImportLog importLog = genericDataImporterService.doDataImport(importConfigurationDs.item, importData)
-        importLogDs.item = importLog
-        toStep5()
+        try {
+            ImportLog importLog = genericDataImporterService.doDataImport(importConfigurationDs.item, importData)
+            importLogDs.item = importLog
+            toStep5()
+        } catch (Exception i) {
+            showNotification("Couldn't bind attributes, please modify entity attributes and try again.", NotificationType.WARNING_HTML)
```

Comment:
Unfortunately a lot of stuff can happen when doing the `doDataImport`. It is not necessarily directly related to a binding problem. I'm not sure if the errors should be hidden to the users at that point in the lifetime of the app component. Eventually this is a good idea, but this is pretty beta right now and it gives the user direct intraction points with the developers.

If we want to hide it, we at least need to log it.

Reply:
Same as previous Exception.

I agree. At least log is necessary.